moped-editor/src/views/projects/newProjectView/DefineProjectForm.js

```diff
@@ -1,102 +1,198 @@
-              required
-              autoFocus
-              label="Project name"
-              name="project_name"
-              type="text"
-              fullWidth
-              value={projectDetails.project_name}
-              error={nameError}
-              helperText="Required"
-              InputLabelProps={{ required: false }}
-              onChange={(e) => handleFieldChange(e.target.value, e.target.name)}
-            />
-          )}
-          {fromSignalAsset && ( // else
-            <Box sx={{ marginBottom: "2.1rem" }}>
-              <SignalAutocomplete
-                signal={signal}
-                setSignal={setSignal}
-                projectDetails={projectDetails}
-                setProjectDetails={setProjectDetails}
-                setSignalRecord={setSignalRecord}
-                signalError={signalError}
+    <Box
+      component="form"
+      onSubmit={handleSubmit(handleSave)}
+      sx={{ paddingX: 9, paddingY: 9 }}
+    >
+      <Grid container spacing={6}>
+        <Grid item xs={12}>
+          <Grid container spacing={3}>
+            <Grid item sm={6} xs={12}>
+              {!isSignalProject && (
+                <ControlledTextInput
+                  autoFocus
+                  variant="standard"
+                  fullWidth
+                  id="project_name"
+                  label="Project name"
+                  name="projectName"
+                  control={control}
+                  error={!!errors?.projectName}
+                  helperText={errors?.projectName?.message}
+                  InputProps={{
+                    disabled: loading,
+                  }}
+                />
+              )}
+              {isSignalProject && (
+                <Controller
+                  id="signal"
+                  name="signal"
+                  control={control}
+                  shouldUnregister={true}
+                  render={({ field }) => (
+                    <KnackComponentAutocomplete
+                      {...field}
+                      componentLabel="Signal"
+                      signalType={"TRAFFIC"}
+                      socrataEndpoint={SOCRATA_ENDPOINT}
+                      isOptionEqualToValue={getSignalOptionSelected}
+                      getOptionLabel={getSignalOptionLabel}
+                      textInputOptions={{
+                        variant: "standard",
+                        error: !!errors?.signal,
+                        helperText: errors?.signal?.message,
+                        disabled: loading,
+                      }}
+                    />
+                  )}
+                />
+              )}
+            </Grid>
+            <Grid item sm={6} xs={12} alignContent="flex-end">
+              <Controller
+                name="isSignalProject"
+                control={control}
+                render={({ field: { onChange, value } }) => (
+                  <FormControlLabel
+                    label="Use signal asset"
```

Comment:
This label is slightly different because its former location above the switch was throwing off the grid alignment.